app/assets/stylesheets/recipes.scss

```diff
@@ -28,6 +28,60 @@
     border: 0;
     margin-top: 1rem;
   }
+
+  input#recipe_archived {
+    width: 30px;
+    margin-left: 20px;
+  }
```

Comment:
This was to fix the giant box that it was. Not related to the other stuff, just wanted to try to fix it when I saw it.